src/_includes/macros/sub-nav.html

```diff
@@ -19,38 +19,43 @@
    ========================================================================== #}
 
 {% macro render(sub_nav_title, sub_nav_landing, sub_nav_items, sub_nav_media) %}
-<div class="sub-nav_wrapper">
-    <div class="sub-nav_inner-wrapper
-                wrapper">
+<div class="sub-nav js-sub-nav">
+    <div class="wrapper wrapper__sub-nav">
+        <button class="sub-nav_btn__back js-sub-nav_back">Back</button>
         <section class="sub-nav_menu">
-            <h4 class="sub-nav_title">
-                <a href="{{ sub_nav_landing }}">{{ sub_nav_title }}</a>
-            </h4>
-            <hr class="content-line content-line__nav u-show-on-desktop">
+            <span class="sub-nav_title">
+                <a class="sub-nav_link" href="{{ sub_nav_landing }}">
+                    {{ sub_nav_title }}
+                </a>
+            </span>
+            <div class="content-l content-l__full">
             {%- for sub_nav_item in sub_nav_items %}
                 <ul class="sub-nav_menu-list
-                           list__unstyled
-                           list-expanding
-                           list-expanding_child-list">
+                           content-l_col
+                           content-l_col-1-3
+                           list__unstyled">
```

Comment:
That brings up a thought. Why do we have a list class and not just style the default `ul, ol, dl` elems?